fix(runner): load credentials before creating a composite root

ostridm opened the pull request:
closes #139

ostridm committed:
fix(runner): add ability to load credentials in `Create` method

closes #139

derevnjuk commented on src/SecTester.Runner/SecRunner.cs:
```diff
@@ -45,8 +45,13 @@ public async ValueTask DisposeAsync()
-  public static SecRunner Create(Configuration configuration)
+  public static async Task<SecRunner> Create(Configuration configuration)
   {
+    if (configuration.Credentials is null)
```
`LoadCredentials` is safe, you can remove this `if`

derevnjuk commented on test/SecTester.Runner.Tests/SecRunnerTests.cs:
```diff
@@ -31,12 +33,43 @@ public SecRunnerTests()
   }
 
+  [Fact]
+  public async Task Create_AbleToLoadCredentials_CreatesCompositeRoot()
```
```suggestion
  public async Task Create_CouldLoadCredentials_CreatesCompositeRoot()
```

derevnjuk reviewed (changes requested):
Please update README accordingly

ostridm committed:
fix(runner): add ability to load credentials in `Create` method

adjust according to code review
https://github.com/NeuraLegion/sectester-net/pull/140#discussion_r1049982033
https://github.com/NeuraLegion/sectester-net/pull/140#discussion_r1049982524

closes #139

derevnjuk commented:
@ostridm PR is still not ready for review. You have to update README https://github.com/NeuraLegion/sectester-net/pull/140#pullrequestreview-1219715694
